### DISCUSSION TITLE
Disabling a feed?

### DISCUSSION THREAD

**Discussion by chemical1979** [FreshRSS/FreshRSS#5581] [Q&A] [answered]
I checked the menus, but apparently I'm unable to disable a feed. Please give me a quick hint.

I want to disable a feed, but not deleting it (I want to remember this feed). I could make this feed "not visible" in its settings, but it still gets updated in the background, correct? As this feed publishes articles on a 15 min basis, this gets annoying pretty fast and also creates a lot of noise in the db.

Maybe I should just delete the feed. But then again ... ;-)

**Comment by Alkarex** [comment]
Hello,
See the *mute* option:

![image](https://user-images.githubusercontent.com/1008324/47867901-2d3c6100-de03-11e8-8a81-f626a121595d.png)

and if you want to hide it:

![image](https://user-images.githubusercontent.com/1008324/47867936-45ac7b80-de03-11e8-9bbf-271d4ffd4dec.png)

**Comment by Alkarex** [comment]
You can also choose a very slow update, such as once a month:
![image](https://user-images.githubusercontent.com/1008324/47868035-81474580-de03-11e8-9cae-0274820db21a.png)

**Comment by chemical1979** [comment]
Muting is exactly what I was looking for. Thank you!

**Comment by notDavid** [comment]
Hi there, so I'm wondering if 'mute' also stops **fetching** the feeds?

If it does, the wording is a bit misleading. Muting implies it is only 'not displayed' and updates still happen in the background.

What I'm looking for is a way to **disable** a feed, so it does not get fetched. It's unclear whether 'mute' accomplishes this, and i can't find anything in the docs.

Does any body know? Thank you!

**Comment by Luxtech** [comment]
> Hi there, so I'm wondering if 'mute' also stops **fetching** the feeds?
> 
> If it does, the wording is a bit misleading. Muting implies it is only 'not displayed' and updates still happen in the background.
> 
> What I'm looking for is a way to **disable** a feed, so it does not get fetched. It's unclear whether 'mute' accomplishes this, and i can't find anything in the docs.
> 
> Does any body know? Thank you!

This! Could somebody clarify?

**Comment by Alkarex** [comment, marked as answer]
To disable a feed, the *mute* option can be used. No new entry will appear anymore. The feed will not be fetched anymore, except if another user also subscribes to that feed.
Muted feeds will appear with a 🔇 prefix in the user interface.
Note that a feed will automatically be set to *muted* if the server returns an *HTTP 410 Gone* response.

**Reply by notDavid** [reply]
Good to know, thank you @Alkarex 

You should consider to add this info to the interface for clarification, or perhaps even better rename the text?
"Mute" seems like a bad label to describe "Disabled".

**Reply by Frenzie** [reply]
Whoops, forgot to submit a PR for that. Feel free to do the honors if you like? Otherwise ping me so I don't forget again. :-D https://github.com/FreshRSS/FreshRSS/issues/2654#issuecomment-554437695
